hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/container/metrics/TestContainerMetrics.java

```diff
@@ -185,7 +174,7 @@ static void runTestClientServer(
       initConf.accept(pipeline);
 
       DatanodeDetails dn = pipeline.getFirstNode();
-      volumeSet = createVolumeSet(dn, TEST_DIR + dn.getUuidString());
+      volumeSet = createVolumeSet(dn, testDir + dn.getUuidString());
```

Comment:
Can you please confirm if `testDir` has `/` at the end?  If it doesn't, this will create a different directory that JUnit does not know about.

Reply:
Good catch. I fixed it.